python/src/VisVersions.py

```diff
@@ -0,0 +1,62 @@
+from enum import Enum
+
```

Comment:
TODO: generer denne fra resources. Det finnes sikkert noe postbuild greier du kan gjøre. Se TS for inspo